python/gui/MaaS/templates/maas/dataset_management.html

```diff
@@ -0,0 +1,797 @@
+            ds_files_content_div.style.height = '90%';
+            ds_files_content_div.style.overflow = 'auto';
+
+            ds_files_div.appendChild(ds_files_content_div);
+
+            let parent_div = document.getElementById('dataset-manage');
+            parent_div.appendChild(ds_files_div);
+            return ds_files_content_div;
+        }
+
+        function ajaxBuildDatasetDetailsTable(dataset_json, table) {
+            buildDetailsTableRow(table, "Name", dataset_json["name"]);
+            buildDetailsTableRow(table, "Category", dataset_json["data_category"])
+            buildDetailsTableRow(table, "Type", dataset_json["type"]);
+            buildDetailsTableRow(table, "Format", dataset_json["data_domain"]["data_format"]);
+            buildDetailsTableRow(table, "Read Only", dataset_json["is_read_only"]);
+            buildDetailsTableRow(table, "Created", dataset_json["create_on"]);
+            buildDetailsTableRow(table, "Last Updated", dataset_json["last_updated"]);
+
+            // TODO: add something for dataset size
+            // TODO: add something for number of files.
+            // TODO: add something for indicating if data aligns properly with configured domain
+
+            // Put spacer row in to indicate the start of domain details
+            let domain_header_row = document.createElement('tr');
+            let colCell = document.createElement('th');
+            colCell.colSpan = 2;
+            colCell.style.textAlign = 'left';
+            colCell.style.paddingTop = '20px';
+            colCell.style.paddingLeft = '5%';
+            colCell.style.fontStyle = 'italic';
+            colCell.style.fontSize = '18px';
+            colCell.appendChild(document.createTextNode("Data Domain"));
+            domain_header_row.appendChild(colCell);
+            table.appendChild(domain_header_row);
+
+            // Then iteratively add domain details rows
+            let c_restrict;
+            for (let c_i = 0; c_i < dataset_json["data_domain"]["continuous"].length; c_i++) {
+                c_restrict = dataset_json["data_domain"]["continuous"][c_i];
+                buildDetailsTableRow(table, c_restrict["variable"], c_restrict["begin"] + " to " + c_restrict["end"]);
+            }
+            let d_restrict;
+            for (let d_i = 0; d_i < dataset_json["data_domain"]["discrete"].length; d_i++) {
+                d_restrict = dataset_json["data_domain"]["discrete"][d_i];
+                buildDetailsTableRow(table, d_restrict["variable"], d_restrict["values"]);
+            }
+        }
+
+        function makeByteSizesReadable(bytes, decimal_places = 2) {
+            if (bytes <= 0) {
+                return "0 B";
+            }
+            let thresh = 1024;
+            let decimals = decimal_places < 0 ? 0 : decimal_places;
+            let units = ['B', 'KB', 'MB', 'GB', 'TB'];
+            let level = Math.floor(Math.log(bytes) / Math.log(thresh))
```

Comment:
`Math.log` is the natural log. I believe we want [`Math.log2`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/log2).

Reply:
nevermind, I see you are doing change of base.